Enterprise-CMCS/eAPD
# NZ/4501 - can add/remove years

Resolves #4501 

### Description
Configure adding/removing years for an MMIS APD.

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| MMIS APD | On APD overview Page |  Can add/remove years | cypress |

### Steps to manually verify this change

1. Go to an MMIS APD overview
2. Add years
3. Remove years
4. Confirm it still works in HITECH APD

### This pull request is ready to code review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when


### This pull request is ready to review when the QA has


### This pull request can be merged when
